src/main/java/fr/insee/rmes/services/datasets/DataSetsImpl.java

```diff
@@ -478,9 +477,18 @@ private List<ThemeModelSwagger> getThemeModelSwaggerS(JSONObject dataSetId) thro
         }
     }
 
+    @NotNull
+    protected List<LangContent> constructLangContent(String elementLg1, String elementLg2) {
```

Comment:
Ecrire une méthode plus synthétique : ` return List.of(LangContent.lg1(elementLg1), LangContent.lg2(elementLg2)` Faudra créer les méthodes dans LangContent

Reply:
ok fait